subsys/bluetooth/host/conn.c

```diff
@@ -1109,6 +1109,10 @@ int bt_conn_set_security(struct bt_conn *conn, bt_security_t sec)
 		return -EOPNOTSUPP;
 	}
 
+	if (IS_ENABLED(CONFIG_BT_SMP_OOB_LEGACY_PAIR_ONLY) &&
+	    sec > BT_SECURITY_L3)
+		return -EOPNOTSUPP;
+
```

Comment:
Missing { }

Reply:
checkpatch complains about style because it is a one line if statement

Reply:
I believe you had other one-liners with {} in this same PR? Anyway, you might get complaints if you run checkpatch yourself without disabling this particular check. The Zephyr coding style however *requires* {} as you'll probably see if you look at any existing Zephyr code. The Zephyr CI checks have this checkpatch feature disabled so you shouldn't get complaints for the PR.

Reply:
I added those {}, and ran checkpatch, I didn't see any warnings from checkpatch.

Reply:
Aha, nvm then will readd